CHANGELOG.md

```diff
@@ -19,6 +19,9 @@ a release.
 
 ## [Unreleased]
 
+### Fixed
+- Tree: Allow multiple entity types with closure strategy (#2653)
```

Comment:
This sentence seems to be describing a feature instead of a bug. Please, try to explain what problem is solved in this PR.